s/swagger_spec_validator/swagger_spec_validator_ubi_8.3.sh

```diff
@@ -0,0 +1,86 @@
+# ----------------------------------------------------------------------------
+#
+# Package               : swagger_spec_validator
+# Version               : 2.7.3
```

Comment:
Is it the only version tested here ? If not other versions tested should also be listed. Rest all LGTM

Reply:
2.7.3 version only tested here